README.md

```diff
@@ -1,4 +1,20 @@
 Put the time in weeks.  We'll change the format as need be.  I'm too busy
 putting my kid to sleep to do this nicely ;)
 
+** Current Format Requirements **
```

Comment:
If you prefix with `##` and omit the surrounding `**` this can be a heading.  You'd also need to add a new line.